docs/release-notes/v1.0.rst

```diff
@@ -1,6 +1,149 @@
 .. _gammapy_1p0_release:
 
-1.0 (unreleased)
-----------------
+1.0 (October 7th, 2022)
+-----------------------
 
-- No changes yet
+Summary
+~~~~~~~
+
+- Released Oct 7th, 2022
+- 10 contributors
+- 78 pull requests since v0.20.1 (not all listed below)
+
+New features
+~~~~~~~~~~~~
+
+This new release is the Long Term Stable (LTS) version 1.0. Most of the changes are in the package
+infrastructure. There are no API-breaking changes introduced since 0.20.1. A number of improvements
```

Comment:
The changes in `LightCurveTemplateTemporalModel` is API breaking. Its not a table on init anymore, users have to go through `.read()`